mempool/nop_mempool.go

```diff
@@ -0,0 +1,113 @@
+package mempool
```

Comment:
can't we move each reactor to different subfolder?

Reply:
I'd prefer to do it in a separate PR.